core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -382,15 +382,45 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     }
   }
 
-  it should "return a successful Submission but no started workflows when given a method configuration with unparseable inputs" in withWorkspaceService { workspaceService =>
-    val submissionRq = SubmissionRequest("dsde", "UnparseableMethodConfig", "Individual", "indiv1", Some("this.sset.samples"), false)
+  it should "400 when given a method configuration with unparseable inputs" in withWorkspaceService { workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "UnparseableInputsMethodConfig", "Individual", "indiv1", Some("this.sset.samples"), false)
+    val rqComplete = intercept[RawlsExceptionWithErrorReport] {
+      Await.result(workspaceService.createSubmission(testData.wsName, submissionRq), Duration.Inf).asInstanceOf[RequestComplete[ErrorReport]]
+    }
+
+    assertResult(StatusCodes.BadRequest) {
+      rqComplete.errorReport.statusCode.get
+    }
+
+    assert {
+      rqComplete.errorReport.message.contains("Invalid inputs: three_step.cgrep.pattern")
+    }
+  }
+
+  it should "400 when given a method configuration with unparseable outputs" in withWorkspaceService { workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "UnparseableOutputsMethodConfig", "Individual", "indiv1", Some("this.sset.samples"), false)
+    val rqComplete = intercept[RawlsExceptionWithErrorReport] {
+      Await.result(workspaceService.createSubmission(testData.wsName, submissionRq), Duration.Inf).asInstanceOf[RequestComplete[ErrorReport]]
+    }
+
+    assertResult(StatusCodes.BadRequest) {
+      rqComplete.errorReport.statusCode.get
+    }
+
+    assert {
+      rqComplete.errorReport.message.contains("Invalid outputs: three_step.cgrep.count")
+    }
+  }
+
+  it should "return a successful Submission when given a method configuration with empty outputs" in withWorkspaceService { workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "EmptyOutputsMethodConfig", "Individual", "indiv1", Some("this.sset.samples"), false)
     val rqComplete = Await.result(workspaceService.createSubmission( testData.wsName, submissionRq ), Duration.Inf).asInstanceOf[RequestComplete[(StatusCode, SubmissionReport)]]
     val (status, newSubmissionReport) = rqComplete.response
     assertResult(StatusCodes.Created) {
       status
     }
 
-    assert( newSubmissionReport.workflows.size == 0 )
+    assert( newSubmissionReport.workflows.size == 3 )
```

Comment:
`newSubmissionReport.workflows should have size 3`

Reply:
I'm going to leave this one since it's the more common style for this file